tests/db_migrator_input/config_db/reclaiming-buffer-traditional-single-pool-expected.json

```diff
@@ -0,0 +1,343 @@
+    },
+    "BUFFER_PORT_EGRESS_PROFILE_LIST|Ethernet24": {
+        "profile_list": "egress_lossless_zero_profile,egress_lossy_zero_profile"
+    },
+    "BUFFER_PORT_INGRESS_PROFILE_LIST|Ethernet20": {
+        "profile_list": "ingress_lossless_zero_profile"
+    },
+    "BUFFER_PORT_EGRESS_PROFILE_LIST|Ethernet16": {
+        "profile_list": "egress_lossless_profile,egress_lossy_profile"
+    },
+    "BUFFER_PORT_EGRESS_PROFILE_LIST|Ethernet20": {
+        "profile_list": "egress_lossless_profile,egress_lossy_profile"
+    },
+    "BUFFER_PORT_INGRESS_PROFILE_LIST|Ethernet16": {
+        "profile_list": "ingress_lossless_profile"
+    },
+    "BUFFER_PORT_INGRESS_PROFILE_LIST|Ethernet24": {
+        "profile_list": "ingress_lossless_profile"
+    },
+    "BUFFER_PROFILE|egress_lossless_profile": {
+        "dynamic_th": "7",
+        "pool": "egress_lossless_pool",
+        "size": "0"
+    },
+    "BUFFER_PROFILE|egress_lossy_profile": {
+        "dynamic_th": "7",
+        "pool": "egress_lossy_pool",
+        "size": "9216"
+    },
+    "BUFFER_PROFILE|ingress_lossless_profile": {
+        "dynamic_th": "7",
+        "pool": "ingress_lossless_pool",
+        "size": "0"
+    },
+    "BUFFER_PROFILE|ingress_lossy_profile": {
+        "dynamic_th": "3",
+        "pool": "ingress_lossless_pool",
+        "size": "0"
+    },
+    "BUFFER_PROFILE|pg_lossless_10000_5m_profile": {
+        "xon": "19456",
+        "dynamic_th": "0",
+        "xoff": "22528",
+        "pool": "ingress_lossless_pool",
+        "size": "19456"
+    },
+    "BUFFER_PROFILE|pg_lossless_40000_5m_profile": {
+        "xon": "19456",
+        "dynamic_th": "0",
+        "xoff": "22528",
+        "pool": "ingress_lossless_pool",
+        "size": "19456"
+    },
+    "BUFFER_PROFILE|pg_lossless_50000_5m_profile": {
+        "xon": "19456",
+        "dynamic_th": "0",
+        "xoff": "22528",
+        "pool": "ingress_lossless_pool",
+        "size": "19456"
+    },
+    "BUFFER_PROFILE|pg_lossless_100000_5m_profile": {
+        "xon": "19456",
+        "dynamic_th": "0",
+        "xoff": "23552",
+        "pool": "ingress_lossless_pool",
+        "size": "19456"
+    },
+    "BUFFER_PROFILE|q_lossy_profile": {
+        "dynamic_th": "3",
+        "pool": "egress_lossy_pool",
+        "size": "0"
+    },
+    "BUFFER_PROFILE|ingress_lossy_pg_zero_profile" : {
+        "pool":"ingress_zero_pool",
+        "size":"0",
+        "static_th":"0"
+    },
+    "BUFFER_PROFILE|ingress_lossless_zero_profile" : {
+        "pool":"ingress_lossless_pool",
+        "size":"0",
+        "dynamic_th":"-8"
+    },
+    "BUFFER_PROFILE|egress_lossless_zero_profile" : {
+        "pool":"egress_lossless_pool",
+        "size":"0",
+        "dynamic_th":"-8"
+    },
+    "BUFFER_PROFILE|egress_lossy_zero_profile" : {
+        "pool":"egress_lossy_pool",
+        "size":"0",
+        "dynamic_th":"-8"
+    },
+    "BUFFER_QUEUE|Ethernet0|0-2": {
```

Comment:
admin down ports are Ethernet20 and Ethernet24. Why are zero profiles expected on other ports which are admin up and not these?

Reply:
Not providing admin status also means admin down.

Reply:
Most of them don't have zero profiles attached. like either BUFFER_PG or BUFFER_QUEUE objects for admin down ports are not having zero profiles attached. I was expecting only items attached to port Ethernet28 to have regular profiles in this file. the rest all should be pointing to zero profiles since those ports either have explicity admin shut or 'admin_status' missing.

Reply:
If the buffer table content is the default on an admin down port, it will be migrated to zero profile.
for BUFFER_PG, the default configuration is empty or only lossless PG which matches the speed and cable length (in case the DUT ran 201811)
for all other tables, the default configuration is empty.
some ports even are admin down, we don't set them to zero profiles since the configuration is not the default.

Reply:
For BUFFER_PG
|port|speed|cable length|lossless pg|lossy pg|default|
|:---:|:---:|:---:|:---:|:---:|:---|
|ethernet0|||None|none|yes|
|ethernet4|10000|5m|pg_lossless_10000_5m_profile|none|yes|
|ethernet8|25000|5m|pg_lossless_10000_5m_profile|none|no|
|ethernet12||||exist|no|
|ethernet16||||exist|no|
|ethernet20|100000|5m|pg_lossless_100000_5m_profile|none|yes|
|ethernet24|||none|none|yes|

for the ports that have a `yes` in `default` column, the zero profile will be applied on lossy PGs and lossless PGs will be kept untouched. buffer manager will handle the lossless PGs.

I think it's easy to check whether the port configuration is default for other tables.